From bb28f64233ebcf18cc0b8837f311ac5f7a731498 Mon Sep 17 00:00:00 2001
From: Stephan Linz <linz@li-pro.net>
Date: Wed, 28 Oct 2015 19:40:02 +0100
Subject: [PATCH] Add comments for Robotron SCP format definitions

Signed-off-by: Stephan Linz <linz@li-pro.net>
---
 diskdefs | 29 ++++++++++++++++++++++++-----
 1 file changed, 24 insertions(+), 5 deletions(-)

diff --git a/diskdefs b/diskdefs
index 630317a..d080091 100644
--- a/diskdefs
+++ b/diskdefs
@@ -367,7 +367,12 @@ diskdef fdd3000_2
   skew 5
 end
 
-# Robotron 1715
+############################
+# GdR/Robotron scp disks
+############################
+
+# robotron PC1715 SCP 185k (40*1* 5,1024 6 OFS 2k DIR) 5.25"
+# setfdprm /dev/fd1 ds dd ssize=1024 cyl=40 sect=5 head=1 dtr=1 fm=0
 diskdef 1715
   seclen 1024
   tracks 40
@@ -379,7 +384,8 @@ diskdef 1715
   os 2.2
 end
 
-# Robotron 1715 with SCP3
+# robotron PC1715 SCP3 760k (80*2* 5,1024 8 OFS 4k DIR) 5.25"
+# setfdprm /dev/fd1 ds dd ssize=1024 cyl=80 sect=5 head=2 dtr=1 fm=0
 diskdef 17153
   seclen 1024
   tracks 160
@@ -391,7 +397,8 @@ diskdef 17153
   os 3
 end
 
-#DDR
+# robotron PC1715 SCP 624k (80*2* 16,256 8 OFS 4k DIR) 5.25"
+# setfdprm /dev/fd1 ds dd ssize=256 cyl=80 sect=16 head=2 dtr=1 fm=0
 diskdef scp624
   seclen 256
   tracks 160
@@ -399,10 +406,13 @@ diskdef scp624
   blocksize 2048
   maxdir 128
   skew 0
-  boottrk 2
+  boottrk 4
   os 2.2
+  libdsk:format scp640
 end
 
+# robotron PC1715 SCP 640k (80*2* 16,256 0 OFS 4k DIR) 5.25"
+# setfdprm /dev/fd1 ds dd ssize=256 cyl=80 sect=16 head=2 dtr=1 fm=0
 diskdef scp640
   seclen 256
   tracks 160
@@ -412,8 +422,11 @@ diskdef scp640
   skew 0
   boottrk 0
   os 2.2
+  libdsk:format scp640
 end
 
+# robotron PC1715 SCP 780k (80*2* 5,1024 8 OFS 4k DIR) 5.25"
+# setfdprm /dev/fd1 ds dd ssize=1024 cyl=80 sect=5 head=2 dtr=1 fm=0
 diskdef scp780
   seclen 1024
   tracks 160
@@ -421,10 +434,13 @@ diskdef scp780
   blocksize 2048
   maxdir 128
   skew 0
-  boottrk 2
+  boottrk 4
   os 2.2
+  libdsk:format scp800
 end
 
+# robotron PC1715 SCP 800k (80*2* 5,1024 0 OFS 4k DIR) 5.25"
+# setfdprm /dev/fd1 ds dd ssize=1024 cyl=80 sect=5 head=2 dtr=1 fm=0
 diskdef scp800
   seclen 1024
   tracks 160
@@ -434,8 +450,11 @@ diskdef scp800
   skew 0
   boottrk 0
   os 2.2
+  libdsk:format scp800
 end 
 
+# robotron Z9001 800k (80*2* 5,1024 0 OFS 6k DIR) 5.25"
+# setfdprm /dev/fd1 ds dd ssize=1024 cyl=80 sect=5 head=2 dtr=1 fm=0
 diskdef z9001
   seclen 1024
   tracks 160
